src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, AreaCalculator {
+    protected String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("No figure to draw");
+    }
```

Comment:
The `Figure` class should not be directly implementing interfaces that have unrelated methods. Consider separating the concerns by having different interfaces for different purposes (e.g., one for calculating area and another for drawing).